<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqliteQtDialog</class>
 <widget class="QDialog" name="SqliteQtDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SqliteQtDialog</string>
  </property>
  <widget class="QComboBox" name="CondComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentText">
    <string>ID</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>score</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ValueComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>升序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>降序</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="idlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>学生学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学生姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="scorelabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>学生成绩：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>301</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="scoreEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>CondComboBox</zorder>
  <zorder>ValueComboBox</zorder>
  <zorder>idlabel</zorder>
  <zorder>namelabel</zorder>
  <zorder>scorelabel</zorder>
  <zorder>insertButton</zorder>
  <zorder>deleteButton</zorder>
  <zorder>modifyButton</zorder>
  <zorder>sortButton</zorder>
  <zorder>idEdit</zorder>
  <zorder>nameEdit</zorder>
  <zorder>scoreEdit</zorder>
  <zorder>tableView</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
